### PR TITLE
(PUP-4026) Add Windows paths created during installlation

### PR DESCRIPTION
**Pull request by joshcooper**

### CODE REVIEW

File: file_paths.md [hunk 1]
```diff
@@ -21,9 +21,12 @@ The files annotated by an '*' indicate that they are created by package installa
     /etc/puppetlabs *                                                    n/a
 
     /etc/puppetlabs/code *                # :codedir                     contents moved from :confdir
-        environments                      # :environmentpath
+        environments *                    # :environmentpath
+          production *
```
Comment (melissa) [inline file_paths.md]: If https://github.com/puppetlabs/puppet-agent/pull/86 gets merged, we should add `environment.conf` to this.

File: file_paths.md [hunk 2]
```diff
@@ -132,22 +135,25 @@ user or group.
...
 
-    C:\ProgramData\PuppetLabs\code            # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
-        environments                          # :environmentpath
-        hiera.yaml                            # :hiera_config
-        hieradata                             # n/a
+    C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
+        environments *                        # :environmentpath
+          production *
+            manifests *
+            modules *
```
Comment (melissa) [inline file_paths.md]: We also might want to add `environment.conf` here

File: file_paths.md [hunk 3]
```diff
@@ -155,7 +161,7 @@ user or group.
         custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
         device.conf                           # :deviceconfig
         fileserver.conf                       # :fileserverconfig
-        puppet.conf                           # :config
+        puppet.conf *                         # :config
```
Comment (melissa) [inline file_paths.md]: I'm not seeing `mcollective/etc/puppet.conf`...

File: file_paths.md (hunk 1 above)
Comment (joshcooper) [inline file_paths.md]: updated

File: file_paths.md (hunk 2 above)
Comment (joshcooper) [inline file_paths.md]: updated based on https://github.com/puppetlabs/puppet_for_the_win/commit/5170b59e61f0862a60b09fc0c77d6c8054f57c8b#diff-d5da35a313714cb36d3944cb9219314fR367

File: file_paths.md [hunk 4]
```diff
@@ -132,124 +136,129 @@ user or group.
...
     C:\ProgramData                                                           n/a
 
-    C:\ProgramData\PuppetLabs\code            # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
-        environments                          # :environmentpath
-        hiera.yaml                            # :hiera_config
-        hieradata                             # n/a
+    C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
+        environments *                        # :environmentpath
+          production *
+            environment.conf *
```
Comment (joshcooper) [inline file_paths.md]: @melissa lost my comments, but I added this.

**Comment by joshcooper** [comment]
@Iristyle @haus @melissa I think this is good

File: file_paths.md [hunk 5]
```diff
@@ -132,124 +136,129 @@ user or group.
...
+            binder_config.yaml                # :binder_config
+            csr_attributes.yaml               # :csr_attributes
+            custom_trusted_oid_mapping.yaml   # :trusted_oid_mapping_file
+            device.conf                       # :deviceconfig
+            fileserver.conf                   # :fileserverconfig
+            puppet.conf *                     # :config
+            routes.yaml                       # :route_file
+            ssl                               # :ssldir
+        var *
+            log                               # :logdir                      same
```
Comment (joshcooper) [inline file_paths.md]: comment from https://github.com/joshcooper/puppet-specifications/commit/6fd4098708192fb8bcf68664c78adaca8f1a9203#commitcomment-10033811, var is created but log is not.

File: file_paths.md [hunk 6]
```diff
@@ -132,124 +136,129 @@ user or group.
...
+        cfacter.bat *                         # bat file wrappers
+        environment.bat *                     # setup LOAD_PATH
+        facter.bat *
+        hiera.bat *
+        mco.bat *
+        puppet.bat *
+        puppet_shell.bat *                    # targets for shortcuts
+        run_cfacter_interactive.bat *
+        run_facter_interactive.bat *
+        run_puppet_interactive.bat *
```
Comment (joshcooper) [inline file_paths.md]: Comment from https://github.com/joshcooper/puppet-specifications/commit/6fd4098708192fb8bcf68664c78adaca8f1a9203#commitcomment-10033853, added individual bat files.

File: file_paths.md [hunk 7]
```diff
@@ -132,124 +136,129 @@ user or group.
...
+    C:\Program Files\Puppet Labs\Puppet\hiera *
+        bin *
+            hiera *                           # ruby bin wrapper
+        lib *
+            hiera.rb *
+
+    C:\Program Files\Puppet Labs\Puppet\mcollective *
+        bin *
+            mcollectived *                    # ruby bin wrapper
+            mco *
```
Comment (joshcooper) [inline file_paths.md]: Comment from https://github.com/joshcooper/puppet-specifications/commit/6fd4098708192fb8bcf68664c78adaca8f1a9203#commitcomment-10033878

File: file_paths.md [hunk 8]
```diff
@@ -132,124 +136,129 @@ user or group.
...
+
+    C:\Program Files\Puppet Labs\Puppet\misc *
+        LICENSE.rtf *                         # license
+        puppetlabs.ico *                      # icon for start menu shortcut
+        puppetres.dll *                       # event log message resource dll
+        versions.txt *                        # versions of components
+
+    C:\Program Files\Puppet Labs\Puppet\puppet *
+        bin *
+            puppet *                          # ruby bin wrapper
```
Comment (joshcooper) [inline file_paths.md]: Comment from https://github.com/joshcooper/puppet-specifications/commit/6fd4098708192fb8bcf68664c78adaca8f1a9203#commitcomment-10033906, left out extlookup2hiera

**Pull request merged**